packages/components/src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -1,11 +1,43 @@
 import classNames from 'classnames';
-import React from 'react';
+import React, { ReactElement, ReactNode, RefObject } from 'react';
 import { CSSTransition } from 'react-transition-group';
-import Dropzone from 'react-dropzone';
+import Dropzone, { DropzoneOptions } from 'react-dropzone';
 import { truncateFileName } from '@deriv/shared';
 import Text from '../text';
 
-const FadeInMessage = ({ is_visible, color, children, key, timeout, no_text }) => (
+type TFadeInMessage = {
+    is_visible: boolean;
+    color?: string;
+    key?: string;
+    timeout: number;
+    no_text?: boolean;
+};
+
+type TPreviewSingle = {
+    dropzone_ref: RefObject<HTMLElement>;
+} & TFileDropzone;
+
+type TFileDropzone = {
+    className?: string;
+    noClick?: boolean;
+    validation_error_message: (open?: () => void) => ReactNode | null;
```

Comment:
```suggestion
    validation_error_message: ReactNode | string | null;
```
please also add the call for ValidationErrorMessage() in **uploader.jsx**:
`validation_error_message={value?.errors?.length ? ValidationErrorMessage**()** : null}`